soccer/src/soccer/strategy/agent/position/defense.cpp

```diff
@@ -164,6 +190,26 @@ std::optional<RobotIntent> Defense::state_to_task(RobotIntent intent) {
     } else if (current_state_ == MARKING) {
         // Marker marker = Marker((u_int8_t) robot_id_);
         return marker_.get_task(intent, last_world_state_, this->field_dimensions_);
+    } else if(current_state_ == STEALING) { //wall stealer
+        auto collect_cmd = planning::MotionCommand{"collect"};
+        intent.motion_command = collect_cmd;
+        intent.dribbler_speed = 255.0;
+        return intent;
+    } else if(current_state_ == CLEARING) {  
+        planning::LinearMotionInstant target{clear_point_};
+        auto line_kick_cmd = planning::MotionCommand{"line_kick", target};
+        intent.motion_command = line_kick_cmd;
+
+        // note: the way this is set up makes it impossible to
+        // shoot on time without breakbeam
+        // TODO(Kevin): make intent hold a manip msg instead? to be cleaner?
```

Comment:
i think it's about time to stop adding TODOs for @kfu02, lol. 

i know you just copied this from somewhere but let's take this out, and maybe make a clickup card if we think this task is actually a good idea